server/src/server.ts

```diff
@@ -0,0 +1,71 @@
+import express from 'express';
+import http from 'http';
+import mongoose from 'mongoose';
+import { config } from './config/config';
+import Logging from './library/logging';
+import settlementRoutes from './routes/settlementRoutes';
+import survivorRoutes from './routes/survivorRoutes';
+
+const router = express();
+
+// Connect to MongoDB
+mongoose
+    .connect(config.mongo.url, { retryWrites: true, w: 'majority' })
```

Comment:
Mongoose is probably handling the close on backend, but good to check anyway just in case